README.md

```diff
@@ -0,0 +1,195 @@
+Send build artifacts to a windows share (CIFS/SMB/samba)
+
+# Publish Over ...
+
+Read Publish Over ... wiki page first
+
+Please read [Publish
+Over](http://localhost:8085/display/JENKINS/Publish+Over) first.  
+Common configuration options that are documented in the Publish Over ...
+wiki page will not be repeated here.
+
+[Features](http://localhost:8085/display/JENKINS/Publish+Over+CIFS+Plugin#PublishOverCIFSPlugin-features)  
+[Configure](http://localhost:8085/display/JENKINS/Publish+Over+CIFS+Plugin#PublishOverCIFSPlugin-configuration)  
+[Change
+log](http://localhost:8085/display/JENKINS/Publish+Over+CIFS+Plugin#PublishOverCIFSPlugin-changelog)
+
+# Features
+
+-   Send files to windows shares
+-   The plugin is "promotion aware" see
+    [Promotions](http://localhost:8085/display/JENKINS/Publish+Over#PublishOver-promotions)
+-   Publish to multiple shares in the "Post-build Actions"
+-   Passwords are encrypted in the configuration files and in the UI
+-   Copy files to a windows share during a build
+
+# Configure
+
+#### Configure a WINS server
+
+From the Jenkins home page, click "Manage Jenkins" and the click on
+"Configure System"
+
+In the main Jenkins configuration page, there will be a "Publish Over
+CIFS" checkbox in the "Global properties" section.
+
+![cifs_node_props_global](docs/images/cifs_node_props_global.png)
+
+###### WINS server
+
+Set this option to the IP address of a WINS server that will be used by
+the Jenkins master, and will be the default for all other nodes (slaves)
```

Comment:
We're using the term "controller" instead of "master" and "agent" instead of "slave".  More inclusive phrasing.
```suggestion
the Jenkins controller, and will be the default for all other agents.
```